src/styles/utils/variables.scss

```
@@ -0,0 +1,8 @@
+$color_main_accent: #060b35;
+$color_blue_accent: #00acdc;
```

It is better to use a more universal naming.

```suggestion
$color_second_accent: #00acdc;
```